### PR TITLE
Casey2

### PR DESCRIPTION
**Pull request by hisayohorie**
- update the councillors data for 2016 election
- added end_date for <a href ="https://www.casey.vic.gov.au/council/news-publications/mediareleases/beardon9mar17">Steve Beardon's resignation </a>

### CODE REVIEW

File: vic_local_councillor_popolo.json [hunk 1]
```diff
@@ -8691,6 +8780,21 @@
       "role": "mayor"
     },
     {
+      "person_id": "casey_city_council/susan_serey",
+      "organization_id": "legislature/casey_city_council",
+      "role": "Dupety Mayor"
```
Comment (equivalentideas) [inline vic_local_councillor_popolo.json]: Typo here, should be 'Deputy'.

Also there's three of them. Is that right?

**Comment by equivalentideas** [comment]
I've assigned this back to you @hisayohorie let me know when I should check again 👍

File: vic_local_councillor_popolo.json (hunk 1 above)
Comment (hisayohorie) [inline vic_local_councillor_popolo.json]: ah! the multiple Deputy Mayors are issue similar to issue#102. There are 2 of them currently, and one from the past

Comment (equivalentideas) [inline vic_local_councillor_popolo.json]: That makes sense 👍 

For the one who's gone, follow what @henare here https://github.com/openaustralia/australian_local_councillors_popolo/pull/103 :

> I also just removed the mayor positions as we don't have a way of marking these as stopping and starting (see #102) and we don't use them at the moment anyway.

**Comment by hisayohorie** [comment]
starting a new branch!

**Pull request closed without merging**